addons/index.md

```diff
@@ -7,16 +7,108 @@ layout: documentation
+
+```sh
+feature:list | grep ^openhab
+```
+
+This will give you a list of all available add-ons.
+It could probably look similar to this example:
+
+```text
+...
+openhab-transformation-xslt               | 0.9.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | XSLT Transformation
+openhab-voice-mactts                      | 0.9.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | MacOS Text-to-Speech
+openhab-binding-amazondashbutton          | 2.0.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | Amazon Dash Button Binding
+openhab-binding-astro                     | 2.0.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | Astro Binding
+openhab-binding-autelis                   | 2.0.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | Autelis Binding
+openhab-binding-avmfritz                  | 2.0.0.SNAPSHOT   |          | Uninstalled | openhab-aggregate-xml   | AVM Fritz!Box Binding
+...
+openhab-binding-network                   │ 2.2.0            │          │ Uninstalled │ openhab-addons-2.2.0    │ Network Binding
+...
+```
+
+According to the [naming convention for bundles]({{base}}/administration/bundles.html#naming-convention-for-bundles) the *id* for our example is *network*.
+
+Another way of getting the correct id, is the url from the add-ons documentation page.
+For example the url for the [mqtt binding]({{base}}/addons/bindings/mqtt1/readme.html) is
+
+````text
+https://docs.openhab.org/addons/bindings/mqtt1/readme.html
+```
+
+So the id would be *mqtt1* in this case.
+Did you notice the trailing *1* in this id?
+This is because the mqqt binding is a 1.x add-on.
+
+> All 1.x add-ons **must** have a trailing 1 in their id.
+
+With this information we can now edit the *addons.cfg* file in the `config/services` folder on the machine you are running openHAB on.
+The path is depending on your installation.
+You can find out the correct locations on the corresponding documentation pages, e.g. [linux]({{base}}/installation/linux.html#file-locations) or [windows]({{base}}/installation/windows.html#file-locations).
+
+The file could look like this (depending on your choosen package and already installed add-ons):
+
+```text
+package = standard
+ui = basic,paper,habpanel
+action = pushover
+binding = astro,mqtt1
+transformation = jsonpath
+persistence = influxdb
+misc = restdocs
+```
+
+To install the network binding like we want in this example, we just need to add the id *network* to the binding section.
+
+```text
+binding = astro,mqtt1,network
+```
+
+After saving the file, the add-on will be installed.
+
+
+### With a `.jar` file
+
+> Attention:
+> This option is adressed to advanced users.
+> Installing add-os with a `.jar`file can lead to problems, because add-on dependencies may not be installed.
+> Please make sure to use this option only in special cases (like add-on testing for an upcoming version) or when you know what you are doing.
```

Comment:
This is also nicely summarized here, if you want to compare: https://community.openhab.org/t/frequently-asked-questions-faqs/17727

I'm missing the reason *why* a user might end up using this method.

Reply:
An important reason to mention this is to facilitate testing and - according to the link above:

> The main reference for openHAB 2 is the documentation at http://docs.openhab.org

:-)

The leading warning in lines 104++ address the drawbacks.